tf2onnx/tfonnx.py

```diff
@@ -345,6 +345,8 @@ def tensorflow_onnx_mapping(g, ops_mapping):
                 m_ops, unm_ops, body_exceptions = tensorflow_onnx_mapping(b_g, ops_mapping)
                 mapped_op += m_ops
                 unmapped_op += unm_ops
+                # topological_sort on the body in case processing has changed the order
+                b_g.topological_sort(b_g.get_nodes())
                 exceptions.extend(body_exceptions)
```

Comment:
Sorting is done twice (once at line 341) -- are both required?

Reply:
The 2nd sorting is **after** processing which might change the order.